packages/plugin-flex/src/sub-commands/flex-plugin.ts

```diff
@@ -180,6 +183,15 @@ export default class FlexPlugin extends baseCommands.TwilioClientCommand {
     }
 
     this.exit = exit;
+
+    const doubleDashIndex = argv.indexOf('--');
+    this.internalScriptArgs = doubleDashIndex === -1 ? [] : argv.slice(doubleDashIndex + 1);
```

Comment:
nit:
```
const hasInternalArgs = doubleDashIndex === -1;
this.internalScriptArgs = !hasInternalArgs ? [] : argv.slice(doubleDashIndex + 1);
if(hasInternalArgs) ...
```

Reply:
Reverting this - the two if statements are opposite, one is `=== -1` and once is `!== -1`